README.md

```diff
@@ -35,9 +35,13 @@
 <div align="center">
   <img alt="agentgateway UI" src="img/architecture.svg" width="600">
 </div>
-<br> 
+<br>
+
+# Intro to Agentgateway Video
+
+[![Agentgateway Intro Video](https://img.youtube.com/vi/SomP92JWPmE/hqdefault.jpg)](https://youtu.be/SomP92JWPmE)
 
-**Key Features:**
+## Key Features:
```

Comment:
This should not be nested under "Intro video".

Can we put the video under `Documentation` maybe?